site/docs/recipes/service-discovery.md

```diff
@@ -4,6 +4,21 @@ sidebar_position: 1
 date: "2023-05-07"
 ---
 
+## tl;dr
+
+For any exposed port in your compose configurations:
+
+```yaml
+services:
+     service_name:
+          ...
+          ports: exposed_port:internal_port
+```
+
+Preevy will generate the following environment variable:
+
+`PREEVY_BASE_URI_{{ comopse_service_name }}_{{ exposed_port }}`
```

Comment:
Here is a shorter TL;DR, see if it works for you:

If the external URL is needed in the compose file (e.g, a BASE_URL environment variable), preevy will supply an environment variable which you can [interpolate](https://docs.docker.com/compose/compose-file/12-interpolation/) as follows:

```yaml
services:
     my_service:
          ...
          env:
            BASE_URL=${PREEVY_BASE_URI_MY_SERVICE_8000}
          ports:
            - 8000
```